src/Microsoft.TemplateEngine.Core/Operations/ConditionalType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.TemplateEngine.Core.Operations
+{
+    // The commonly used conditional types. If more get added to ConditionalConfig.cs, they should be added here too.
+    public enum ConditionalType
```

Comment:
(No change required) The C-Style/XML-style evaluators will be provider-ified soon - if there are any considerations to take in to account for their configuration (which provider/any config/etc) we should keep them in mind